examples/suites/main_test.go

```diff
@@ -0,0 +1,21 @@
+package suites
```

Comment:
missing boilerplate header

Reply:
Weird, ./hack check did not catch that one.